metaflow/metaflow_config.py

```diff
@@ -467,7 +467,7 @@ def get_pinned_conda_libs(python_version, datastore_type):
                 DEBUG_OPTIONS.extend(o)
                 for typ in o:
                     vars()["DEBUG_%s" % typ.upper()] = from_conf(
-                        "DEBUG_%s" % typ.upper()
+                        "DEBUG_%s" % typ.upper(), False
```

Comment:
can this be just `bool(from_conf("DEBUG_%s" % typ.upper()))`?

Reply:
No -- there is special handling in the from_conf function that handles a boolean default value (ie: it does the conversion from "0", "false" to False so it needs to be a boolean default.